lib/liquid.rb

```diff
@@ -85,3 +85,5 @@ module Liquid
 #
 Dir["#{__dir__}/liquid/tags/*.rb"].each { |f| require f }
 Dir["#{__dir__}/liquid/registers/*.rb"].each { |f| require f }
+
+require 'liquid/legacy'
```

Comment:
Final newline missing here:
```suggestion
require 'liquid/legacy'

```

Reply:
👍 ... when you github merge quickly